Initial tables (#1)
turbot/steampipe-plugin-auth0: docs/tables/auth0_signing_key.md

# Table: auth0_signing_key

Signing keys are used to sign ID tokens, access tokens, SAML assertions, and WS-Fed assertions sent to your application or API.

## Examples

### For how long has current signing key been available

```sql
select
  current_date - current_since as current_for
from
  auth0_signing_key
where
  current;
```

### Next signing key

```sql
select
  kid,
  fingerprint,
  thumbprint
from
  auth0_signing_key
where
  next;
```

### Previous signing key

```sql
select
  kid,
  fingerprint,
  thumbprint,
  current_since,
  current_until
from
  auth0_signing_key
where
  previous;
```

> ### Average time previous signing key had been available

Reviewer: ```suggestion
### Average time for which the previous signing keys were available
```
